frontend-react/src/components/MessageTracker/MessageReceivers.tsx

```diff
@@ -178,11 +180,14 @@ const MessageReceiversRow = ({
             <td className={getColumnClasses(ColumnDataEnum.Main)}>
                 <p
                     className={classnames(
-                        "font-mono-sm border-1px bg-primary-lighter radius-md padding-left-1 padding-right-1 margin-top-0",
+                        "font-mono-sm padding-left-1 padding-right-1 margin-top-0",
                         {
-                            "bg-blue-5": uppercaseMain === StatusEnum.BATCH,
-                            "bg-blue-10": uppercaseMain === StatusEnum.PROCESS,
-                            "bg-blue-20": uppercaseMain === StatusEnum.READY,
+                            "bg-blue-5 border-1px bg-primary-lighter radius-md":
```

Comment:
Not asking for this now, but it might be nice to split this out into like a StatusPill component or something.  Let's keep tabs on this and see if we'll actually use this motif elsewhere for now!